packages/java/hilla-react/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.13.1</version>
+            <version>4.13.2</version>
```

Comment:
As the dependency is already in the root pom, can the version be removed here? That's how other nested modules are configued. 

Also the module has no tests at all, so the dependency could also be removed I guess.

Reply:
Good, removed. JUnit 4  should not be used for anything new either